department-of-veterans-affairs/caseflow
# Adding Advanced on Docket UI

Resolves #6257

### Description
This renames the `advance_on_docket_grants` to `advance_on_docket_motions`, it creates a UI to grant/deny these motions, and a backend to receive those requests.

We also create a higher ordered component called EditModalBase which enables us to quickly create modals that edit values on the appeal. It's the modal equivalent of DecisionReviewBase.

### Testing Plan
1. Go to case details for an AMA appeal
1. Start the AOD editing flow, and grant AOD
1. Ensure the appeal is marked as AOD
